frameworks/elastic/tests/test_sanity.py

```diff
@@ -160,6 +159,9 @@ def test_xpack_toggle_with_kibana(default_populated_index):
     doc = config.get_document(config.DEFAULT_INDEX_NAME, config.DEFAULT_INDEX_TYPE, 2, service_name=foldered_name)
     assert doc["_source"]["name"] == "X-Pack"
 
+    # reset upgrade strategy to serial
+    config.update_app(foldered_name, {'UPDATE_STRATEGY': 'serial'}, current_expected_task_count)
```

Comment:
Can you add a "wait for deployment / update / recovery" to be complete here? Don't want next test to get tripped up.

Reply:
This already happens inside `config.update_app` [here](https://github.com/mesosphere/dcos-commons/blob/6f184e7d410023087914a297fdae6162299f2c5e/frameworks/elastic/tests/config.py#L184). And then `pre_test_setup` runs before the next test.

Reply:
👍